augly/audio/intensity.py

```diff
@@ -201,3 +201,9 @@ def time_stretch_intensity(rate: float = 1.5, **kwargs) -> float:
 
 def to_mono_intensity(metadata: Dict[str, Any], **kwargs) -> float:
     return 0.0 if metadata["src_num_channels"] == 1 else 100.0
+
+
+def loop_audio_intensity(n: int = 1, **kwargs) -> float:
+    assert isinstance(n, int) and n >= 1, "n must be a positive integer"
```

Comment:
you'll need to change this assert to match the one in `functional.py` (also it would be great if you can change this func name/reorder this function to be in alphabetical order, similar to the suggestion i made in `functional.py`)